airflow/www/views.py

```diff
@@ -154,6 +154,18 @@ def truncate_task_duration(task_duration):
     return int(task_duration) if task_duration > 10.0 else round(task_duration, 3)
 
 
+def sanitize_args(args: dict[str, str]) -> dict[str, str]:
+    """
+    Remove all parameters starting with `_`
+    :param args: arguments of request
+    :return: the dictionary passed as input with args starting with `_` removed.
+    """
+    for key in list(args.keys()):
+        if key.startswith("_"):
+            del args[key]
+    return args
```

Comment:
This mutates args in place which might be unexpected/cause some unforeseen issues down the line so:

```suggestion
    return {
        key: value
        for key, value in args.items()
        if not key.startswith("_")
    }
```

Reply:
Yeah. I was pondering on that one too.. Will change.